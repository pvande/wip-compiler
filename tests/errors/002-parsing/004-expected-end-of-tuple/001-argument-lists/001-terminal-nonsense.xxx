x := (a : int, garbage garbage garbage) => {}
