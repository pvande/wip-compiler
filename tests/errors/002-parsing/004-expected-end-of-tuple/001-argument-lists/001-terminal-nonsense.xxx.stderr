Error: "Unexpected code in argument list"
In [1;37mtests/errors/002-parsing/004-expected-end-of-tuple/001-argument-lists/001-terminal-nonsense.xxx[0m on line [1;37m1[0m

> [0;36mx := (a : int, garbage garbage garbage) => {}[0m
  [0;31m              ^^^^^^^^^^^^^^^^^^^^^^^^^      [0m
